cmake/branding/recolor_graphics.sh.in

```diff
@@ -26,11 +25,13 @@ fi
 
 echo "$BUILD_TYPE build type detected!"
 
+# shellcheck disable=SC2157
 if [ -n "@LMMS_BUILD_APPLE@" ]; then
 	SIZES=(16 32 64 128 256 512)
 	LOCATION="lmms.iconset/icon_RESxRES.png"
 	HIRES="lmms.iconset/icon_RESxRES@2x.png"
 	mkdir recolored/lmms.iconset
+# shellcheck disable=SC2157
```

Comment:
According to the [shellcheck wiki page](https://github.com/koalaman/shellcheck/wiki/Directive), there shouldn't be a shellcheck directive for `else`. The one for `if` will apply to the `else` condition as well.

Reply:
If there's one, `shellcheck` fails.